server/server_test.go

```diff
@@ -12,205 +12,325 @@ import (
+
+			It("should initialize OPA policy", func() {
+				// ignore Grafeas list occurrences call
+				grafeasClient.EXPECT().ListOccurrences(gomock.Any(), gomock.Any()).AnyTimes()
 
-			It("should list Grafeas occurrences", func() {
-				// expect Grafeas list occurrences call
-				grafeasClient.EXPECT().ListOccurrences(gomock.AssignableToTypeOf(context.Background()), listOccurrencesRequest)
+				// expect OPA initPolicy call
 
 				_, _ = rodeServer.AttestPolicy(context.Background(), attestPolicyRequest)
 			})
 
-			When("Grafeas list occurrences response is ok", func() {
+			When("OPA policy initializes", func() {
 
-				It("should evaluate OPA policy", func() {
-					// mock Grafeas list occurrences response
-					listOccurrencesResponse := &grafeas_proto.ListOccurrencesResponse{
-						Occurrences: []*grafeas_proto.Occurrence{
-							createRandomOccurrence(grafeas_common_proto.NoteKind_NOTE_KIND_UNSPECIFIED),
-						},
-					}
-					grafeasClient.EXPECT().ListOccurrences(gomock.Any(), gomock.Any()).Return(listOccurrencesResponse, nil)
-
-					// expect evalute OPA policy call
+				It("should list Grafeas occurrences", func() {
+					// expect Grafeas list occurrences call
+					grafeasClient.EXPECT().ListOccurrences(gomock.AssignableToTypeOf(context.Background()), listOccurrencesRequest)
 
 					_, _ = rodeServer.AttestPolicy(context.Background(), attestPolicyRequest)
 				})
 
-				When("resource does not have previous attestation occurrence", func() {
-					BeforeEach(func() {
-						// mock Grafeas list occurrences response
-						// mock OPA evalute policy response
-					})
-					It("should create new attestation occurrence", func() {
-						// expect Grafeas create occurrence call
-						// _, _ = rodeServer.AttestPolicy(context.Background(), attestPolicyRequest)
-					})
-					It("should respond with new attestation occurrence", func() {
-						// expect response state to match OPA policy evaluation
-						// expect response to include new attestation
-						// _, _ = rodeServer.AttestPolicy(context.Background(), attestPolicyRequest)
-					})
-				})
+				When("Grafeas list occurrences response is ok", func() {
 
-				When("resource has previous attestation occurrence", func() {
-					BeforeEach(func() {
+					It("should evaluate OPA policy", func() {
 						// mock Grafeas list occurrences response
+						listOccurrencesResponse := &grafeas_proto.ListOccurrencesResponse{
+							Occurrences: []*grafeas_proto.Occurrence{
+								createRandomOccurrence(grafeas_common_proto.NoteKind_NOTE_KIND_UNSPECIFIED),
+							},
+						}
+						grafeasClient.EXPECT().ListOccurrences(gomock.Any(), gomock.Any()).Return(listOccurrencesResponse, nil)
+
+						// expect evalute OPA policy call
+
+						_, _ = rodeServer.AttestPolicy(context.Background(), attestPolicyRequest)
 					})
-					When("OPA policy evaluation is same as previous attestation occurrence", func() {
+
+					When("resource does not have previous attestation occurrence", func() {
```

Comment:
there's a decent amount of commented code after this point.  is that needed, or can it be removed?

Reply:
the comments are placeholders for work to be completed